crates/biome_js_analyze/tests/specs/nursery/noEmptyBlockStatements/invalid.js

```diff
@@ -0,0 +1,69 @@
+function foo() {}
+
+const bar = () => {};
+
+function fooWithNestedEmptyFnBlock() {
+  let a = 1;
+
+  function shouldFail(){}
+
+  return a
+}
+
+
+const barWithNestedEmptyFnBlock = () => {
+  let a = 1;
+
+  const shouldFail = () => {}
+
+  return a
+}
+
+let someVar;
+if (someVar) {
+}
+
+while (someVar) {
+}
+
+switch(someVar) {
+}
+
+try {
+    doSomething();
+} catch(ex) {
+
+} finally {
+
+}
+
+class Foo {
+  static {}
+}
+
+for(let i; i>0; i++){}
```

Comment:
Last thing before merging: do you think it makes sense adding an exception for all loop statements?

Reply:
Just my two cents. I'm not opposed to adding an exception if you think it's appropriate.

Reply:
Let's ship the rule as is. We see later if users complain about the default behavior.